srcpkgs/grub/template

```diff
@@ -50,6 +50,13 @@ do_configure() {
 	unset CC AS LD RANLIB CPP
 	CFLAGS="$CFLAGS -fno-stack-protector"
 
+	# building with altivec generates broken grub core
+	case "$XBPS_TARGET_MACHINE" in
+		ppc64*)
+			CFLAGS="$CFLAGS -mno-altivec"
```

Comment:
This could be on one line  `ppc64*) CFLAGS+=" -mno-altivec";;` like in many other places. xbps-src requires bash.